src/encoded/schemas/note_interpretation.json

```
@@ -51,11 +51,11 @@
             "description": "Significance classification indicated by this note",
             "type": "string",
             "enum": [
-                "benign",
-                "likely benign",
+                "Benign",
+                "Likely Benign",
                 "VUS",
-                "likely pathogenic",
-                "pathogenic"
+                "Likely Pathogenic",
+                "Pathogenic"
```

I'm fine with the above, but the [ACMG guidelines](https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf) (page 10) don't capitalize the second word and also don't use "VUS", using "Uncertain significance" instead. My instinct would be to just use those exactly, and it'd be consistent with the candidacy naming style of capitalization, but I haven't been in the loop for previous conversations, so there may be reasons for deviating.
```suggestion
                "Benign",
                "Likely benign", 
                "Uncertain significance",
                "Likely pathogenic",
                "Pathogenic"
```

---

Oh yeah this is a good point.